lib/elixir/lib/option_parser.ex

```diff
@@ -492,4 +556,31 @@ defmodule OptionParser do
       :unknown
     end
   end
+
+  defp format_errors(errors, opts) do
+    details = Enum.map_join(errors, ". ", &format_error(&1, opts))
+    total = Enum.count(errors)
+    error_pluralize = pluralize(total, "error", "errors")
+    "#{total} #{error_pluralize} found! #{details}"
+  end
+
+  defp format_error({option, nil}, _) do
+    "Unknown option #{option}"
+  end
+
+  defp format_error({option, value}, opts) do
+    option_key = option |> String.lstrip(?-) |> String.to_atom()
+
+    type =
+      if opt_alias = opts[:aliases][option_key] do
+        opts[:strict][opt_alias]
+      else
+        opts[:strict][option_key]
+      end
```

Comment:
Now that I'm reading this, maybe we can call the `opt_alias` variable `option_alias`? All other variables start with `option_` (except for `opts`, let's make an exception there :stuck_out_tongue:) and it reads weird when some start with `option_` and some with `opt_`. Wdyt? Sorry for giving not-so-good advice before :).